packages/dd-trace/src/plugins/tracing.js

```diff
@@ -81,6 +82,8 @@ class TracingPlugin extends Plugin {
       }
     })
 
+    span.setTag(COMPONENT, this.component)
```

Comment:
This could be set above to avoid an additional call to `setTag`.

Reply:
Changed so that component is set above with the other tags within the startSpan call.